packer/scylla_install_image

```diff
@@ -123,6 +124,7 @@ if __name__ == '__main__':
         sysconfig_opt = '--disable-writeback-cache'
         swap_opt = '--swap-directory /mnt'
 
+    run('systemctl disable apt-daily-upgrade.timer apt-daily.timer dpkg-db-backup.timer motd-news.timer', shell=True, check=True)
```

Comment:
You *may* need to either run 'disable --now' or also 'stop', IIRC!

Reply:
When we create an instance based on our AMI we boot it, so the disabling is enough. also verified it locally when tested the minimal image

Reply:
It's a good practice (and in theory, they might trigger while you are building the AMI)

Reply:
changed